backend/lcfs/db/seeders/common/notifications_seeder.py

```diff
@@ -58,47 +58,124 @@ async def seed_notification_types(session):
     """
     types_to_seed = [
         {
-            "name": "TRANSFER_PARTNER_UPDATE",
-            "description": "Transfer partner update notification",
-            "email_content": "Email content for transfer partner update",
+            "name": "BCEID__CR__DIRECTOR_ASSESSMENT",
+            "description": "Director assessment",
```

Comment:
Please add more descriptive information, like "Director assessed compliance report"

Reply:
Sure, I’ve updated it with more descriptive keys and detailed descriptions.